webapp/locales/de.json

```diff
@@ -91,9 +91,9 @@
     }
   },
   "admin": {
-    "name": "Systemverwaltung",
+    "name": "Admin",
     "dashboard": {
-      "name": "Startzentrale",
+      "name": "Dashboard",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-07T22:41:58Z" title="Saturday, June 8th 2019, 12:41:58 am +02:00">Jun 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -91,9 +91,9 @@
     }
   },
   "admin": {
-    "name": "Systemverwaltung",
+    "name": "Admin",
     "dashboard": {
-      "name": "Startzentrale",
+      "name": "Dashboard",
```


Yes I think most Germans understand the meaning of "Dashboard" by now. Literally translated it means "Armaturenbrett".

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-06-08T05:07:47Z" title="Saturday, June 8th 2019, 7:07:47 am +02:00">Jun 8, 2019</time>_

---


certain words just don't work in other languages. we are forced to find our own terms. 
in the admin area it has more to do with people who can use the word dashboard. and expect something. 
linguistics is definitely an important branch and our cultural burden in europe :) 
i think it's important that we develop our own style. one world .. you can think if people are one step ahead.